src/iap/shared/components/game-card/game-card.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="card">
```

Comment:
:point_right: :point_left: 
```suggestion
<div class="iap-card">
```